app/routes/administrative-units/administrative-unit/change-events/details/index.js

```diff
@@ -0,0 +1,66 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+import { CHANGE_EVENT_TYPE } from 'frontend-contact-hub/models/change-event-type';
+
+export default class AdministrativeUnitsAdministrativeUnitChangeEventsDetailsIndexRoute extends Route {
+  @service store;
+
+  async model() {
+    let administrativeUnit = this.modelFor(
+      'administrative-units.administrative-unit'
+    );
+
+    let { changeEventId } = this.paramsFor(
+      'administrative-units.administrative-unit.change-events.details'
+    );
+    let changeEvent = await this.store.findRecord(
+      'change-event',
+      changeEventId,
+      {
+        reload: true,
+        include: [
+          'type',
+          'decision',
+          'original-organization',
+          'resulting-organization',
+          'results.resulting-organization',
+          'results.status',
+        ].join(),
+      }
+    );
+
+    let currentChangeEventResult = await findCurrentChangeEventResult(
+      administrativeUnit,
+      changeEvent
+    );
+
+    let changeEventType = await changeEvent.type;
+
+    return {
+      administrativeUnit,
+      changeEvent,
+      currentChangeEventResult,
+      isMergerChangeEvent: changeEventType.id === CHANGE_EVENT_TYPE.MERGER,
+      shouldShowExtraInformationCard:
+        shouldShowExtraInformationCard(changeEventType),
```

Comment:
It's probably better to move these to the controller since they are derived state.